### PR TITLE
Add lodestar beaconcha.in node metrics support

### PR DESCRIPTION
**Pull request by nflaig**
Template changes for https://github.com/rocket-pool/smartnode/pull/335

Related https://github.com/rocket-pool/smartnode-install/pull/71

Requires Lodestar v1.8.0 (release date 17/18 Apr.)

### CODE REVIEW

File: install/scripts/start-bn.sh [hunk 1]
```diff
@@ -138,6 +138,15 @@ if [ "$CC_CLIENT" = "lodestar" ]; then
         CMD="$CMD --checkpointSyncUrl $CHECKPOINT_SYNC_URL"
     fi
 
+    if [ "$ENABLE_BITFLY_NODE_METRICS" = "true" ]; then
+        CMD="$CMD --monitoring.endpoint $BITFLY_NODE_METRICS_ENDPOINT?apikey=$BITFLY_NODE_METRICS_SECRET&machine=$BITFLY_NODE_METRICS_MACHINE_NAME"
+
+        # monitoring relies on metrics data
+        if [ "$ENABLE_METRICS" != "true" ]; then
```
Comment (nflaig) [inline install/scripts/start-bn.sh:145]: As monitoring relies on metrics data, it is required that metrics are also enabled by supplying the `--metrics` flag. This is no longer required on https://github.com/ChainSafe/lodestar/pull/5328 is merged an released, so might just wait for this before merging changes in this PR.

Comment (jclapis) [reply install/scripts/start-bn.sh:145]: Ok, I'll merge it once the next Lodestar release is out.

Comment (nflaig) [reply install/scripts/start-bn.sh:145]: Perfect, I just realized that the UX could really be improved when creating this PR putting, marking as draft until then

**Comment by nflaig** [comment]
[v1.8.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.8.0) has been released ✔️

**Comment by nflaig** [comment]
Should this be merged as well? The other PR that depends on those changes has been merged already
- https://github.com/rocket-pool/smartnode/pull/335

**Comment by jclapis** [comment]
> Should this be merged as well? The other PR that depends on those changes has been merged already
> 
>     * [Enable lodestar beaconcha.in node metrics support smartnode#335](https://github.com/rocket-pool/smartnode/pull/335)

Yep, forgot about this guy - merging now. Thanks!

**Pull request merged**